changeset:   206:f3e86f91fe83
tag:         tip
user:        Alexandre Rostovtsev <tetromino@gentoo.org>
date:        Tue Apr 02 09:39:53 2013 -0400
files:       gtkspell/Makefile.am
description:
s/GTK_SPELL_CFLAGS/GTK_SPELL3_CFLAGS/

Fixes build failure with some gobject-introspection versions; see
https://bugs.gentoo.org/show_bug.cgi?id=463674


diff -r 2079cc186a81 -r f3e86f91fe83 gtkspell/Makefile.am
--- a/gtkspell/Makefile.am	Mon Mar 04 21:54:20 2013 +0100
+++ b/gtkspell/Makefile.am	Tue Apr 02 09:39:53 2013 -0400
@@ -40,7 +40,7 @@
 if ENABLE_GTK3
 GtkSpell-3.0.gir: libgtkspell3-3.la
 GtkSpell_3_0_gir_INCLUDES = GObject-2.0 GLib-2.0 Gtk-3.0
-GtkSpell_3_0_gir_CFLAGS = --warn-all @GTK_SPELL_CFLAGS@
+GtkSpell_3_0_gir_CFLAGS = --warn-all @GTK_SPELL3_CFLAGS@
 GtkSpell_3_0_gir_LIBS = libgtkspell3-3.la
 GtkSpell_3_0_gir_FILES = $(libgtkspell3_3_la_SOURCES)
 GtkSpell_3_0_gir_SCANNERFLAGS = --c-include gtkspell/gtkspell.h

